prometheus.yml

```diff
@@ -0,0 +1,22 @@
+
+
+# Sample Prometheus config
+# This assumes that your Prometheus instance can access this application on localhost:8080
+
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+  external_labels:
+    monitor: 'codelab-monitor'
+rule_files:
+  - 'prometheus.rules.yml'
+scrape_configs:
+  - job_name: 'spring-backend'
+    metrics_path: '/actuator/prometheus'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:8080']
```

Comment:
will the local host also work on Azure?

Reply:
This is old, to work on azure it needs the service name instead of localhost. It is changed, I first did the prometheus only for backend and tested without using the docker image to be able to find the the spring app I had to put my pc ip adress instead of localhost. then I changed to put the service name "spring" and pushed. I now made a new pull request on deployment I added prometheus and grafana to deployment docker-compose.yml  so that we can access from azure , let me know if something fishy

Reply:
Ignore this to commits, I made a right pull request and merged myself :) next time I will wait for review, but it was on point only the grafana ports I suspect you changed.